src/components/Auth/Login/Login.react.js

```diff
@@ -62,11 +62,10 @@ class Login extends Component {
     super(props);
     this.state = {
       email: '',
-      password: '',
-      success: false,
-      validForm: false,
```

Comment:
@isuruAb Dear sir, only the validForm state variable has been removed, and the rest have been reordered. The reason for removing validForm is because we don't need it to be stored as a state variable. Instead, we could simply calculate it while rendering the component, making the whole system simpler ( Lesser state variables to deal with ).